deploy/dracon/chart/values.yaml

```diff
@@ -56,13 +56,12 @@ deduplication-db-migrations:
   # if set, a Job will be deployed that applies migrations to the deduplication database
   # the Job will run as part of the post-install/post-upgrade hook
   enabled: true
-  database:
-    host: ""
-    auth:
-      username: ""
-      password: ""
-      database: ""
-      querystringargs: ""
+  database: *psqlConfig
```

Comment:
this looks odd, the postgresql section should accept an auth section, not a database section https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml#L32

Reply:
why not use the anchor for the auth dictionary of the database and pass that to the auth section of the postgresql configuration?

Reply:
`auth` does not include the `host` attribute however

Reply:
```yaml

# this section controls the database that components have access to
# the database should use the Postgres dialect.
# the database is used by the deduplication enricher
database: &psqlConfig
  host: "aaaaaaa"
  auth:
    username: "bla"
    password: ""
    database: ""
    postgresPassword: ""
    querystringargs: ""

postgresql:
  # if set, a PostgreSQL instance will be deployed
  enabled: true
  <<: *psqlConfig

# this section controls aspects of managing a database used to store deduplication enrichments
# the database should use the Postgres dialect.
deduplication-db-migrations:
  # if set, a Job will be deployed that applies migrations to the deduplication database
  # the Job will run as part of the post-install/post-upgrade hook
  enabled: true
  database: *psqlConfig

image:
  # registry to use for all 
  registry: ""
```
after offline discussions